inference-engine/src/transformations/src/transformations/common_optimizations/dropout_with_random_uniform_replacer.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/dropout_with_random_uniform_replacer.hpp"
+
+#include <memory>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include "itt.hpp"
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::DropoutWithRandomUniformReplacer, "DropoutWithRandomUniformReplacer", 0);
+
+ngraph::pass::DropoutWithRandomUniformReplacer::DropoutWithRandomUniformReplacer() {
+    MATCHER_SCOPE(DropoutWithRandomUniformReplacer);
+    const auto data_pattern = ngraph::pattern::any_input();
+    const auto shape_of_pattern = ngraph::pattern::wrap_type<opset8::ShapeOf>({data_pattern});
```

Comment:
You don't use data_pattern later. We can remove it:
```suggestion
    const auto shape_of_pattern = ngraph::pattern::wrap_type<opset8::ShapeOf>();
```